bolt/lib/RuntimeLibs/InstrumentationRuntimeLibrary.cpp

```diff
@@ -57,12 +57,6 @@ void InstrumentationRuntimeLibrary::adjustCommandLineOptions(
               "the input binary\n";
     exit(1);
   }
-  if (!BC.FiniFunctionAddress && !BC.IsStaticExecutable) {
```

Comment:
While you're here may I ask you to move the statement "if (BC.IsStaticExecutable && !opts::InstrumentationSleepTime) {" from emitBinary here ? It seems to be more proper place. Thanks!

Reply:
I don't mind moving it there (and I agree it's more logical) but I feel it would be better to do this in a separate PR as it's not really related to this one.